katharsis-spring/src/main/java/io/katharsis/spring/boot/KatharsisRegistryConfiguration.java

```diff
@@ -51,4 +52,23 @@ public ExceptionMapperRegistry exceptionMapperRegistry() throws Exception {
         ExceptionMapperRegistryBuilder mapperRegistryBuilder = new ExceptionMapperRegistryBuilder();
         return mapperRegistryBuilder.build(properties.getResourcePackage());
     }
+
+
+    @Bean
+    public ServiceUrlProvider getServiceUrlProvider() {
+        return new ServiceUrlProvider() {
+
+            @Value("${katharsis.pathPrefix}")
+            private String pathPrefix;
+
+            @Resource
+            private HttpServletRequest request;
+
+            public String getUrl() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)